src/shop/_queries/useGetRewards.ts

```diff
@@ -0,0 +1,82 @@
+import { usePubliqApi } from '../../_hooks/usePubliqApi';
+import { TApiError } from '../../_http';
+import { useGetMe } from '../../profile/_queries/useGetMe';
+import { TRewardCategory, TRewardsResponse, TRewardType } from '../_models/reward';
+
+export type TFilterRewardCategory = TRewardCategory | 'laatste kans';
+export type TFilterRewardSections = 'online' | 'in de kijker' | 'populair regio' | 'populair' | 'stad voordelen' | 'sport';
+
+export function useGetRewards({
+  category,
+  type = 'POINTS',
+  section,
+  itemsPerPage = 20,
+  extraParams = {},
+}: {
+  category?: TFilterRewardCategory;
```

Comment:
kan je maar 1 categorie meegeven?

Reply:
Voorlopig wel... er is nergens een optie om meerdere categorieen te combineren